apisix/cli/etcd.lua

```diff
@@ -179,14 +179,14 @@ function _M.init(env, show_output)
 
             local post_json_auth = dkjson.encode(json_auth)
             local response_body = {}
-            local _, err = http.request{url = auth_url, method = "POST",
+            local res, err = http.request{url = auth_url, method = "POST",
                                         source = ltn12.source.string(post_json_auth),
                                         sink = ltn12.sink.table(response_body),
                                         headers = {["Content-Length"] = #post_json_auth}}
             -- In case of failure, request returns nil followed by an error message.
             -- Else the first return value is just the number 1
```

Comment:
The first return value isn't the response body?

Reply:
Yep. It has two forms.
You can see more details about it.

>In case of failure, the function returns nil followed by an error message. If successful, the simple form returns the response body as a string, followed by the response status code, the response headers, and the response status line. The generic function returns the same information, except the first return value is just the number 1 (the body goes to the sink).

http://w3.impa.br/~diego/software/luasocket/http.html

Reply:
Got it